src/server/w32_svr_periodic_hook.c

```diff
@@ -419,6 +456,7 @@ main(int argc, char *argv[])
 	pbs_loadconf(0);
 	/* initialize the pointers in the resource_def array */
 
+	(void)snprintf(path_hooks_workdir, MAXPATHLEN, "%s/server_priv/hooks/tmp/",pbs_conf.pbs_home_path);
```

Comment:
I don't think that on windows path can be given in this form. remember it uses '\' on windows. This will apply to all the paths that this binary is dealing with.
We should probably pass these as command line arguments to the windows binary and prepare these paths in pbs_server itself.

Reply:
Thanks for catching, I will modify this.

Reply:
Ah, WinAPI handles this!
File I/O functions in the Windows API convert "/" to "\" as part of converting the name to an NT-style name. [Ref.] (https://docs.microsoft.com/en-us/windows/desktop/FileIO/naming-a-file)